doc/doxygen/headers/glossary.h

```diff
@@ -974,7 +974,7 @@
  *
  * The full reference for this paper is as follows:
  * @code
-Article{BK07,
+@Article{BK07,
```

Comment:
Nope, not inside a code region, see https://www.dealii.org/developer/doxygen/deal.II/DEALGlossary.html#distributed_paper